src/throttle.decorator.ts

```diff
@@ -19,3 +19,18 @@ export const Throttle = (limit = 20, ttl = 60): MethodDecorator & ClassDecorator
     return target;
   };
 };
+
+export const SkipThrottle = (): MethodDecorator & ClassDecorator => {
```

Comment:
I was thinking we could have
```ts
export const SkipThrottle = (skip = true) => {
  Reflect.defineMetadata(THROTTLER_SKIP, skip, target)
}
```
So that a class can overall be skipped, but a route can still be throttled.